Explain how to configure Renovate for GitLab (#255)
redhat-appstudio/docs.appstudio.io: docs/modules/ROOT/pages/concepts/RenovateBot/index.adoc

RenovateBot monitors Tekton bundle references and makes sure that they're up to date. When new versions of the Tekton bundles are available, RenovateBot creates a pull request with the updated references. This gives you time to review, change, and incorporate the new bundles.

== Authentication and authorization

> To create pull requests in target source code repositories Renovate bot has to have proper credentials.

Reviewer: ```suggestion
To create pull requests in the target source code repositories Renovate bot must have proper credentials.
```

>  * GitHub Application - Requires `Red Hat Konflux` GitHub App to be installed in the source code repository.

Reviewer: ```suggestion
 * **GitHub Application** - Requires `Red Hat Konflux` GitHub App to be installed in the source code repository.
```

>  * GiLab. Create a link:https://docs.renovatebot.com/modules/platform/gitlab/#authentication[Personal Access Token] for the bot account.  Add PAT using link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_creating-secrets-for-your-builds/[

Reviewer: ```suggestion
 * **GiLab application**. Create a link:https://docs.renovatebot.com/modules/platform/gitlab/#authentication[Personal Access Token] (PAT) for the Renovate bot account.  Add PAT using link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_creating-secrets-for-your-builds/[
```

Reviewer: It says - "Add PAT using..." Where to add this?  ....  in Gitlab?

> Creating secrets for your builds]  The secret has to have such labels:

Reviewer: ```suggestion
Creating secrets for your builds].  The secret has to have such labels:
```

Reviewer: When we say "such labels" are these just examples? or these are the labels that user should have?

In case these are examples, I believe we can update the sentence to: The secret should have labels, for example:

If these are something a secret must have: The secrets must have following labels:

Reviewer: The secret must have the following labels:

+
* `appstudio.redhat.com/credentials: scm`
* `appstudio.redhat.com/scm.host: <your_domain_name> - For example,  appstudio.redhat.com/scm.host: gitlab.myhost.com
